thunderbolt/thunderbolt.py

```diff
@@ -58,6 +58,19 @@ def get_task_df(self, all_data: bool = False) -> pd.DataFrame:
             return df
         return df[['task_id', 'task_name', 'last_modified', 'task_params']]
 
+    def get_data(self, task_name: str) -> Union[list, Any]:
+        """Load newest task output data.
+
+        Args:
+            task_name: gokart's task name.
+
+        Returns:
+            The return value is newest data or data list.
+        """
+        df = self.get_task_df()
+        df = df.sort_values(by='last_modified', ascending=False)
+        return self.load(df.query(f'task_name=="{task_name}"')['task_id'].iloc[0])
```

Comment:
IMO, It might be fantastic if we can control recent "k" items rather than only recent one, isn't it ?

Reply:
It's great idea! Let's make an issue!
We will respond with another PR.